.github/workflows/build.yml

```diff
@@ -0,0 +1,15 @@
+name: build
+
+on:
+  push:
+    branches: [dev]
+    paths-ignore:
+      - "docs/**"
+      - "readme.md"
+
+  pull_request:
+    branches: [dev]

```

Comment:
Can we centralize these? These are pretty much the same for every project of ours too. Same for HE.

Reply:
Not as far as I can tell with github actions.

The triggers, i.e. on, need to be in the source repo

Reply:
Also `paths-ignore`?

Reply:
Pathe ignore is just an on: argument so same would apply I think

Reply:
Please check if there are any other options.

Reply:
There are no options for centralizing these

Reply:
I see, sad. The good thing is that these are very unlikely do change.